.github/workflows/ci-br.yml

```diff
@@ -74,12 +75,15 @@ jobs:
         run: curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
       - name: start tikv cluster
         run: |
-          # start tikv in apiv1ttl
+          # start tikv
+          echo -e "\napi-version = ${{ matrix.api_version }}\n" >> /home/runner/work/migration/migration/.github/config/br_rawkv.toml
           /home/runner/.tiup/bin/tiup playground ${{ matrix.tikv_version }} --mode tikv-slim --kv 1 --without-monitor --kv.config /home/runner/work/migration/migration/.github/config/br_rawkv.toml --pd.config /home/runner/work/migration/migration/.github/config/br_pd.toml &> raw.out 2>&1 &
           # The first run of `tiup` has to download all components so it'll take longer.
           sleep 1m 30s
           # Parse PD address from `tiup` output
           echo "PD_ADDR=$(cat raw.out | grep -oP '(?<=PD client endpoints: \[)[0-9\.:]+(?=\])')" >> $GITHUB_ENV
+          # Output the api version
+          echo "API_VERSION=${{ matrix.api_version }}" >> $GITHUB_ENV
           # Log the output
           echo "$(cat raw.out)" >&2
       - name: run integration test
```

Comment:
Suggest to add a step to print BR log after this. Whether failpoint is run as expected can only be verified by BR log now.

Reply:
Added.